flomo/cli.py

```diff
@@ -39,5 +40,16 @@ def start(tag: str, name: str):
     ui.main(tag.lower(), name, session_id)
 
 
+@flomo.command(aliases=["t"])
+def tracking():
```

Comment:
can add a command to reset the data (delete the sessions.db file and re-initialise_